docker-compose.yml

```diff
@@ -42,4 +57,4 @@ services:
     command:
       - --name=app
       - --playground
-
+      - --indexer=http://subquery-node:3000
```

Comment:
The usage of `--indexer` has been deprecated.

Reply:
we should mark it as such on yargs as well